testme-intellij-core/build.gradle

```diff
@@ -0,0 +1,98 @@
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+
+apply plugin: 'java'
+apply plugin: 'groovy'
+apply plugin: 'org.jetbrains.intellij.platform'
```

Comment:
shouldn't the child projects use [org.jetbrains.intellij.platform.module](https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-plugins.html#module) plugin instead ? while letting only the root build file use [org.jetbrains.intellij.platform ](https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-plugins.html#platform)plugin - accordingly pluginConfiguration should only be set on the root build file